helion/_compiler/inductor_lowering.py

```diff
@@ -570,6 +639,79 @@ def decorator(handler: CodegenHandler) -> CodegenHandler:
     return decorator
 
 
+def _register_inductor_lowering(
```

Comment:
Lets moving the inductor lowering patching related stuff to a new file.  This one is getting big.